tasks/main.yml

```
@@ -1,2 +1,64 @@
 ---
-# tasks file for skeleton
+- name: Load var file with package names based on the OS type
+  ansible.builtin.include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml"
+        - "{{ ansible_distribution }}.yml"
+        - "{{ ansible_os_family }}.yml"
+      paths:
+        - "{{ role_path }}/vars"
+
+# Check if Nessus Agent is already installed
+- name: Check if Nessus Agent is already installed
+  ansible.builtin.stat:
+    path: "{{ install_directory }}"
+  register: nessus_directory
+
+- name: Install Nessus Agent
+  block:
+    - name: Grab Nessus Agent  system package from S3
+      ansible.builtin.aws_s3:
```

Is this the right FQDN? Per https://docs.ansible.com/ansible/latest/collections/index_module.html, I think it's:

```suggestion
      amazon.aws.aws_s3:
```

---

Fixed in 5c64f77.